tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/PathVerificationTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.testerina.test;
+
+import org.ballerinalang.test.context.BMainInstance;
+import org.ballerinalang.test.context.BallerinaTestException;
+import org.ballerinalang.test.context.LogLeecher;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/**
+ * Test class containing tests related to test path verification.
+ */
+public class PathVerificationTest extends BaseTestCase {
+
+    private BMainInstance balClient;
+    private String projectPath;
+
+    @BeforeClass
+    public void setup() throws BallerinaTestException {
+        balClient = new BMainInstance(balServer);
+        projectPath = outsideTestsProjectPath.toString();
+    }
+
+    @Test
+    public void verifyTestsOutsidePath() throws BallerinaTestException {
+        LogLeecher passingLeecher = new LogLeecher("1 passing");
+        LogLeecher failingLeecher = new LogLeecher("0 failing");
+        balClient.runMain("test", new String[]{"pathVerification.tests"}, null, new String[0],
```

Comment:
Why do we give it as `pathVerification.tests`? Looks like the module structure is wrong. The path to test files seems to be `outside-tests/src/pathVerification.tests` but the correct path should be `outside-tests/src/pathVerification/tests`.

Reply:
@azinneera The path to tests is not wrong. I have both source files and test files since this check is to see whether tests defined in source files are being executed.
I guess the module name is confusing, I will get rid of the ".tests" part.